tensorflow_addons/layers/tests/crf_test.py

```diff
@@ -284,7 +284,7 @@ def test_numerical_accuracy():
     base_model = tf.keras.Model(x_input, crf_outputs)
     model = ModelWithCRFLoss(base_model)
 
-    model.compile()
+    model.compile(optimizer="Adam")
```

Comment:
@seanpmorgan why we should specify adam here?

Reply:
The default optimizer loss scaler had issue with mixed precision. Not related to our repo I believe.